src/back-end/schema.js

```diff
@@ -76,10 +76,10 @@ const userSchema = {
 			objectType: String,
 			parent: String,
 			date: Date,
+			content: [String],
```

Comment:
I just want to make a note that this change means we need to drop the backend again.

Reply:
Let me discuss this with waynar, i might remove it since it makes more sense to implement in your changes rather than mine. But we agreed this would be a better way to store it

Reply:
Yeah I'm taking charge of this refactor. Delete it from here please Ehson, thanks!

Reply:
Will do 
and ill fix linting again